spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'rails_helper'
 
-RSpec.shared_examples 'enrollment_with_a_status_update' do |passed:, status:, response_json:|
+RSpec.shared_examples 'enrollment_with_a_status_update' do |passed:,
+                                                            email_type:,
+                                                            enrollment_status:,
```

Comment:
I added `email_type` as a new parameter and renamed `status` to `enrollment_status`